assets/hip-hts/solidity/HederaResponseCodes.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Please rename the folder from `hip-hts` to `hip-206`

Reply:
done